config_manager.py

```
@@ -0,0 +1,348 @@
+        return None
+
+    def _load_config_yml(self) -> Dict[str, Any]:
+        """Load config.yml file."""
+        if not self.config_yml_path.exists():
+            logger.warning(f"config.yml not found at {self.config_yml_path}")
+            return {}
+
+        try:
+            with open(self.config_yml_path, 'r') as f:
+                return yaml.safe_load(f) or {}
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)